src/lib/support/tests/TestJsonToTlvToJson.cpp

```diff
@@ -703,9 +703,8 @@ void TestConverter_Array_Empty_ImplicitProfileTag4(nlTestSuite * inSuite, void *
     writer.ImplicitProfileId = kImplicitProfileId;
 
     NL_TEST_ASSERT(gSuite, CHIP_NO_ERROR == writer.StartContainer(TLV::AnonymousTag(), TLV::kTLVType_Structure, containerType));
-    NL_TEST_ASSERT(gSuite,
-                   CHIP_NO_ERROR ==
-                       writer.StartContainer(TLV::ProfileTag(kImplicitProfileId, 1000000), TLV::kTLVType_Array, containerType2));
+    NL_TEST_ASSERT(
+        gSuite, CHIP_NO_ERROR == writer.StartContainer(TLV::ProfileTag(0x000Fu, 0, 0x4240u), TLV::kTLVType_Array, containerType2));
```

Comment:
this is not easy to grasp that this ix 1000000. Maybe this should be `(1_000_000 >> 16)` and `1_000_000 % 0xFFFF` in there instead.